src/main/java/com/atlassian/plugins/confluence/markdown/MarkdownFromURLMacro.java

```diff
@@ -232,7 +232,9 @@ public String execute(Map<String, String> parameters, String bodyContent, Conver
                     
             String rendermermaidjs ="<script>\n" +
                     "AJS.$('[data-macro-name=\"markdown-from-url\"] .language-mermaid').each(function(i, block) {\n" +
+					"block.innerHTML = block.innerHTML.replace(/{<!-- -->{/g, \"{{\");\n"+
                     "const config = {\n"+
+					"    securityLevel:'loose',\n"+
```

Comment:
I don't have a better suggestion on the hexagon thing, but the ``securityLevel: loose`` should be set to ``strict`` or ``sandbox`` (https://mermaid.js.org/config/schema-docs/config-properties-securitylevel.html#securitylevel-constraints)

Reply:
Agreed that `sandbox` seems like the right setting for our risk profile.